### ISSUE TITLE
Text height increased without glyph size change in 2.210

### ISSUE THREAD

**Issue by kchibisov** [JetBrains/JetBrainsMono#366]
From 2.200 to 2.210 the font's text height artificially increased by 1px without any update to the glyphs height or anything like that at all.  As you can see the text looks exactly the same as for the height of the glyphs ( I think I've aligned them incorrectly when doing screen shot, but if overlap them they are the same)

![picture](https://user-images.githubusercontent.com/27620401/99243503-d18a7800-2811-11eb-896b-4b9176e4d822.png)

However  the text height is 1px more.

![picture](https://user-images.githubusercontent.com/27620401/99244392-11059400-2813-11eb-8c30-deac32c21ce3.png)

The metric that got changed is height from FreeType. You can read about how it's computed in

https://www.freetype.org/freetype2/docs/reference/ft2-base_interface.html#ft_facerec (look for height there)

This change affecting all applications which are using FreeType (I think mostly everything on Linux).

I see that 1px was even mentioned in changelog for 2.210, but it really seems artificial, since nothing changed for regular glyphs.

**Comment by kchibisov** [comment]
If the height change is intended(since it's not that bad tbqh). The line drawing elements are not connected.

![picture](https://user-images.githubusercontent.com/27620401/99885612-511bab00-2c47-11eb-840a-5e088d9e7cb5.png)


Font version: 2.211, size 10pt.

**Comment by philippnurullin** [comment]
The height of the line was increased due to support of Vietnamese language. Sorry, It was the only way to make it work. The only other solution was to remove the support of Vietnamese language, so i came up with a compromise like this. 
You don't see changes because, because you don't use Vietnamese alphabet. 

The gaps between BoxDrawing elements in a known issue. Because of increased height all of the BD element needs to be made a bit taller. It will be fixed with one of the next updates.

To sum it up. The height won't be made smaller. The BoxDrawing elements will be made bigger.

**Comment by philippnurullin** [comment]
Made a separate issue for BoxDrawimg. Closing this one.
